Makefile

```diff
@@ -23,7 +23,7 @@ RELEASE_DIR ?= out
 export ACK_GINKGO_DEPRECATIONS := 1.16.5
 export ACK_GINKGO_RC=true
 
-PROJECT_DIR := $(shell dirname $(abspath $(lastword $(MAKEFILE_LIST))))
+export PROJECT_DIR := $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
```

Comment:
Our test images don't have abspath installed, so this variable breaks in our tests.

Our paths do have realpath. Otherwise, I think realpath is also more commonly installed.